org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/IOUtils.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4xml.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * IO utilities class
+ * 
+ * @author Angelo ZERR
+ *
+ */
+public class IOUtils {
+
+	/**
+	 * Convert the given input stream in a String.

```

Comment:
Convert the given {@link InputStream} into a String.
The source InputStream will then be closed.